src/components/PokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,43 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+const PokemonCard = ({pokemonName}) => {
+    const [ pokemon, setPokemon] = useState({})
+
+
+    useEffect(() => {
+        async function getPokemonData() {
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`)
+                // console.log(response.data)
+                setPokemon(response.data)
+            } catch (e) {
+                console.error(e)
+            }
+        }
+
+       void getPokemonData()
+    }, [])
+    return (
+        <div>
+        {Object.keys(pokemon).length > 0 &&
```

Comment:
Dan kan deze check wat simpeler
```suggestion
        {pokemon &&
```